parts/k8s/addons/kubernetesmasteraddons-calico-daemonset.yaml

```diff
@@ -127,7 +31,7 @@ data:
           "datastore_type": "kubernetes",
           "nodename": "__KUBERNETES_NODE_NAME__",
           "mtu": 1500,
-          "ipam": <calicoIPAMConfig>,
+          "ipam" : <calicoIPAMConfig>,
```

Comment:
Unneeded whitespace.

Reply:
Thanks @sylr for the review. The manifests has been auto generated for aks-engine from a calico manifest. I will fix the script to output a better format.